intermediate_source/torch_export_tutorial.py

```diff
@@ -7,25 +7,42 @@
 """
 
 ######################################################################
-# :func:`torch.export` is the PyTorch 2.0 way to export PyTorch models into
-# static and standardized model representations, intended
+# :func:`torch.export` is the PyTorch 2 way to export PyTorch models into
```

Comment:
@williamwen42 Can we add a clear enough warning at the top of the tutorial that says torch.export is in prototype status, this tutorial only applies to 2.1 snapshot and there will be BC breaking changes in subsequent versions?

Some of the planned BC breaking changes:
https://github.com/pytorch/pytorch/pull/108448/files/e2f4c3d6ed1e78665a0ad56022099d025f2010a2..3f1fb6710841d57d142c33d99fe183d0ecd15434